tests/kafkatest/services/kafka/templates/log4j.properties

```diff
@@ -136,6 +136,6 @@ log4j.additivity.state.change.logger=false
 log4j.logger.kafka.authorizer.logger={{ log_level|default("DEBUG") }}, authorizerInfoAppender, authorizerDebugAppender
 log4j.additivity.kafka.authorizer.logger=false
 
-#New Group Coordinator logging.
-log4j.logger.org.apache.kafka.coordinator.group={{ log_level|default("DEBUG") }}, kafkaInfoAppender, kafkaDebugAppender
-log4j.additivity.org.apache.kafka.coordinator.group=false
+#Group Coordinator logging.
```

Comment:
nit: can we add a space after the '#' and in L135 to conform with the other comments?